packages/dag4-network/src/dto/v2/transaction.ts

```diff
@@ -16,15 +16,23 @@ type TransactionValueV2 = {
 
 export type TransactionV2 = {
   hash: string
+  ordinal: number
   source: string
   destination: string
   amount: number
   fee: number
-  parent: TransactionReference 
-  snapshot: string
-  block: string
+  parent: TransactionReference
+  salt: number
+  blockHash: string
+  snapshotHash: string
+  snapshotOrdinal: number
+  transactionOriginal: {
```

Comment:
This value is not present on all GL0/ML0 snapshots, please type as nullish => value?: {type} | null.